src/components/utility/amounts/Amount.tsx

```diff
@@ -58,7 +58,7 @@ const Amount: React.FC<AmountProps> = ({
    return (
       <div className='inline-flex items-center'>
          {showUnit === 'prefix' && <span className={unitClassName}>{getUnitSymbol(unit)}</span>}
-         <span className={className}>
+         <span className={`${className} pt-2`}>
```

Comment:
to try and align the unit symbol with the number. Maybe its better to have a `size` prop for this component instead of having other components set the styling?